licensed_pile/compare_data.py

```diff
@@ -0,0 +1,101 @@
+"""Compare pre and post processed dolma examples."""
```

Comment:
More accurate would be "compare dolma examples across different processing stages"?